tests/contract_testcases/substrate/selector_override_inherited.dot

```diff
@@ -1,28 +1,28 @@
-	diagnostic_23 [label="found contract 'child3'\nlevel Debug\ntests/contract_testcases/substrate/selector_override_inherited.sol:17:1-19:2"]
-	diagnostic_24 [label="base function needs same selector as selector 'abcdef02' of function 'func'\nlevel Error\ntests/contract_testcases/substrate/selector_override_inherited.sol:18:2-56"]
-	note_25 [label="location of base function\ntests/contract_testcases/substrate/selector_override_inherited.sol:14:2-32"]
+	contract [label="contract base\ntests/contract_testcases/substrate/selector_override_inherited.sol:1:1-4:2"]
+	func [label="function func\ncontract: base\ntests/contract_testcases/substrate/selector_override_inherited.sol:3:2-32\nsignature func()\nvisibility public\nmutability nonpayable\nvirtual\nselector abcdef01"]
+	contract_3 [label="contract child\ntests/contract_testcases/substrate/selector_override_inherited.sol:6:1-9:2"]
+	base [label="base base\ntests/contract_testcases/substrate/selector_override_inherited.sol:6:19-23"]
+	func_5 [label="function func\ncontract: child\ntests/contract_testcases/substrate/selector_override_inherited.sol:8:2-33\nsignature func()\nvisibility public\nmutability nonpayable\noverride\nselector abcdef02"]
+	contract_6 [label="contract child2\ntests/contract_testcases/substrate/selector_override_inherited.sol:11:1-13:2"]
+	base_7 [label="base base\ntests/contract_testcases/substrate/selector_override_inherited.sol:11:20-24"]
+	func_8 [label="function func\ncontract: child2\ntests/contract_testcases/substrate/selector_override_inherited.sol:12:2-33\nsignature func()\nvisibility public\nmutability nonpayable\noverride"]
+	contract_9 [label="contract base2\ntests/contract_testcases/substrate/selector_override_inherited.sol:15:1-17:2"]
+	func_10 [label="function func\ncontract: base2\ntests/contract_testcases/substrate/selector_override_inherited.sol:16:2-32\nsignature func()\nvisibility public\nmutability nonpayable\nvirtual"]
+	contract_11 [label="contract child3\ntests/contract_testcases/substrate/selector_override_inherited.sol:19:1-22:2"]
+	base_12 [label="base base2\ntests/contract_testcases/substrate/selector_override_inherited.sol:19:20-25"]
+	func_13 [label="function func\ncontract: child3\ntests/contract_testcases/substrate/selector_override_inherited.sol:21:2-33\nsignature func()\nvisibility public\nmutability nonpayable\noverride\nselector abcdef02"]
+	diagnostic [label="found contract 'base'\nlevel Debug\ntests/contract_testcases/substrate/selector_override_inherited.sol:1:1-4:2"]
+	diagnostic_16 [label="found contract 'child'\nlevel Debug\ntests/contract_testcases/substrate/selector_override_inherited.sol:6:1-9:2"]
+	diagnostic_17 [label="selector '[0xab, 0xcd, 0xef, 0x2]' of function 'func' different from base selector '[0xab, 0xcd, 0xef, 0x1]'\nlevel Error\ntests/contract_testcases/substrate/selector_override_inherited.sol:7:2-37"]
```

Comment:
`0x02`?

Reply:
I've removed selector printing from the error. The location points to the selector already